Cargo.lock

```diff
@@ -2241,7 +2259,7 @@ dependencies = [
 name = "oak_stage0"
 version = "0.1.0"
 dependencies = [
- "bitflags 1.3.2",
+ "bitflags 2.3.3",
```

Comment:
This change seems problematic (also the `oak_linux_boot_params` bitflags dependency above). Rather than having to fix the bitflags version in every cargo.toml, perhaps it could be fixed by using `cargo update` with `--precise` to force a specific version for bitflags on the entire workspace.

Reply:
Also, it does not quite work:

```
$ cargo update -p bitflags@2.3.3 --precise 1.3.2
    Updating crates.io index
error: failed to select a version for the requirement `bitflags = "^2.3.0"`
candidate versions found which didn't match: 1.3.2
location searched: crates.io index
required by package `virtio-drivers v0.5.0`
    ... which satisfies dependency `virtio-drivers = "*"` (locked to 0.5.0) of package `oak_restricted_kernel v0.1.0 (/usr/local/google/home/razieh/dev/code/project-oak/oak/oak_restricted_kernel)`
perhaps a crate was updated and forgotten to be re-vendored?
```

Reply:
Created to https://github.com/project-oak/oak/issues/4223, to track fixing the issue the right way.